media-gfx/shotwell/shotwell-0.30.4.ebuild

```diff
@@ -12,7 +12,7 @@ HOMEPAGE="https://wiki.gnome.org/Apps/Shotwell"
 
 LICENSE="LGPL-2.1"
 SLOT="0"
-KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
```

Comment:
You can not do this. Only arch testers / maintainers can. 

I suggest you to not drop the current stable version, instead just make a version bump, then after 30 days if no significant bugs are opened for it, ask for it to be stabilized. When it has been stabilized, old stable can be dropped from the tree.

Reply:
Got it, and fixed, thanks! :)